libs/langchain/langchain/chains/cube/prompt.py

```diff
@@ -0,0 +1,23 @@
+# flake8: noqa
+from langchain.prompts.prompt import PromptTemplate
+
+PROMPT_SUFFIX = """Only use the following meta-information for Cubes defined in the data model:
+{model_meta_information}
+Question: {input}"""
+
+_DEFAULT_TEMPLATE = """Given an input question, first create a syntactically correct Cube query to run, then look at the results of the query and return the answer. Unless the user specifies in his question a specific number of examples he wishes to obtain, always limit your query to at most {top_k} results. You can order the results by a relevant column to return the most interesting examples in the database.
```

Comment:
> You can order the results by a relevant column to return the most interesting examples in the database.

Maybe it is worth to use more specific word than `interesting` to explain LLM its final goal.
Probably replacing `interesting` with `relevant` could be more evident to LLM despite duplicating the words.